### PR TITLE
fix: set isLoading to false after dispatch

### PR DESCRIPTION
**Pull request by liruchen32**
# Issue
[[defect]重要報導回顧-無資料時應正常顯示empty state](https://app.asana.com/0/1203699264568808/1207367216972411/f)

# Notice

# Dependency

### CODE REVIEW

File: src/components/my-reading/reviewing-article-section.js [hunk 1]
```diff
@@ -103,8 +102,8 @@ const ReviewingArticleSection = () => {
       } else {
         setShowingReviewingArticle(formattedReviews)
       }
-      setIsLoading(false)
     })
+    setIsLoading(false)
```
Comment (Aylie-Chou) [inline src/components/my-reading/reviewing-article-section.js:106]: Question: do we need to wait for fetching data done (finally) before setting `isLoading` false ?

**Review by Aylie-Chou — CHANGES_REQUESTED** [review]
as comment ~

File: src/components/my-reading/reviewing-article-section.js (hunk 1 above)
Comment (liruchen32) [reply src/components/my-reading/reviewing-article-section.js]: fixed

**Review by Aylie-Chou — APPROVED** [review]
LGTM

**Pull request merged**